llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -9238,11 +9259,25 @@ LoopVectorizationPlanner::tryToBuildVPlanWithVPRecipes(VFRange &Range) {
          "VPBasicBlock");
   RecipeBuilder.fixHeaderPhis();
 
+  if (Legal->hasUncountableEarlyExit()) {
+    assert(Legal->getUncountableExitingBlocks().size() == 1 &&
```

Comment:
Easier to do `if (auto *UncountableExitingBlock = Legal->getUncountableExitingBlock())`?